lib/commentTreeUtils.es6.js

```diff
@@ -51,30 +47,33 @@ function parseCommentData(data) {
   return comment;
 }
 
-export function normalizeCommentReplies(comments, visitComment) {
-  return _normalizeCommentReplies(comments, visitComment, true);
-}
-
-function _normalizeCommentReplies(comments, visitComment, isTopLevel) {
-  if (!comments.length) { return; }
+const COMMENT_DEFAULTS = {
+  numReplies: 0,
+  loadMoreIds: [],
+  loadMore: false,
+};
 
+export function normalizeCommentReplies(comments, isTopLevel, visitComment) {
   return comments.map(comment => {
     if (comment.type === COMMENT_LOAD_MORE) { return; }
 
+    // assign some helpful keys and their defaults to the comment
+    Object.assign(comment, COMMENT_DEFAULTS);
+
     // Filter out if a comment is a "load more" type, set a property on the
     // parent comment, and then nuke the fake "reply"
-    if (comment.replies) {
```

Comment:
i removed this check as I believe it to be redundant since we check to see all comments at this point should have a replies field.